From 76f5bf3546d826dcbc03acbefcf0b10b972bf136 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 10 Aug 2022 17:19:38 -0700
Subject: [PATCH 2/2] unix.c: Do not redefine DIR as FILE

DIR is already provided on Linux via
/usr/include/dirent.h system header

Upstream-Status: Inactive-Upstream
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 unix/unix.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/unix/unix.c b/unix/unix.c
index ba87614..6e6f4d2 100644
--- a/unix/unix.c
+++ b/unix/unix.c
@@ -61,13 +61,11 @@ local time_t label_utim = 0;
 /* Local functions */
 local char *readd OF((DIR *));
 
-
 #ifdef NO_DIR                    /* for AT&T 3B1 */
 #include <sys/dir.h>
 #ifndef dirent
 #  define dirent direct
 #endif
-typedef FILE DIR;
 /*
 **  Apparently originally by Rich Salz.
 **  Cleaned up and modified by James W. Birdsall.
-- 
2.37.1

